packages/fx-core/src/plugins/resource/apiconnector/questions.ts

```diff
@@ -85,6 +85,38 @@ export const apiLoginUserNameQuestion: TextInputQuestion = {
   ), // Use the placeholder to display some description
 };
 
+export const basicAuthPassword: TextInputQuestion = {
+  name: Constants.questionKey.apiPassword,
+  title: getLocalizedString("plugins.apiConnector.getQuestion.basicAuth.password.title"),
+  type: "text",
+  password: true,
+  placeholder: getLocalizedString(
+    "plugins.apiConnector.getQuestion.basicAuth.password.placeholder"
+  ), // Use the placeholder to display some description
+};
+
+export const appTenantIdQuestion: TextInputQuestion = {
+  name: Constants.questionKey.apiAppTenentId,
+  title: getLocalizedString("plugins.apiConnector.appTenantId.title"),
```

Comment:
Please add the description in feature spec as placeholder

Reply:
It is fixed. Thanks.